lib/dl_api_lib_testing/dl_api_lib_testing/connector/connection_suite.py

```diff
@@ -79,3 +84,34 @@ def test_connection_options(
         assert isinstance(resp.json["options"]["allow_typed_query_usage"], bool)
         assert len(resp.json["options"]["query_types"]) > 0
         assert any([qt["query_type"] == "generic_query" for qt in resp.json["options"]["query_types"]])
+
+    def test_create_connections__query_params_in_db_name__error(
+        self,
+        control_api_sync_client: SyncHttpClientBase,
+        saved_connection_id: str,
+        bi_headers: Optional[dict[str, str]],
+        connection_params: dict,
+    ) -> None:
+        if not self.uses_classic_sql_connection_schema:
```

Comment:
Maybe let's skip the test on this sort of condition?

```py3
if "db_name" not in connection_params:
    pytest.skip("No db_name field in schema")
```

This way we won't need to bother about what schema each connector uses and can throw away the `uses_classic_sql_connection_schema` flag